client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/gateway/user_gateway/IUsersGateway.kt

```diff
@@ -0,0 +1,10 @@
+package org.thechance.common.data.remote.gateway.user_gateway
```

Comment:
the gateway interface must be in domain layer and its implementation must be in data layer you broke the architecture